src/components/Calculator.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import Table from './Table';
+
+export default class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const tableData = [
+      [0],
+      ['AC', '+/-', '%', '+'],
+      [7, 8, 9, `${String.fromCharCode(215)}`],
+      [4, 5, 6, '-'],
+      [1, 2, 3, '+'],
```

Comment:
- [ ] There are duplicate `+` signs there, instead, it should be the `÷` (division) sign in one of the place.

Reply:
Thank you. I was wondering why we have two plus sign:smile:
My bad eye! I greatly appreciate:pray: